src/targets/gpu/fuse_ops.cpp

```diff
@@ -161,6 +169,10 @@ MIGRAPHX_PRED_MATCHER(bias_shape, instruction_ref ins)
 
 MIGRAPHX_PRED_MATCHER(fusable_conv, instruction_ref ins)
 {
+    const auto device_name     = split_string(get_device_name(), ':').front();
+    const auto supported_archs = get_supported_archs();
```

Comment:
Remove this variable, and just use `get_supported_archs()` directly in the contains: `contains(get_supported_archs(), device_name)`.

Reply:
Removed